modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/oauth/ocs/PlatformManagedOAuthConnectionProvider.java

```diff
@@ -235,6 +231,22 @@ private ConnectionProvider<C> createDelegate(PlatformManagedConnectionDescriptor
     }, MuleException.class, e -> e);
   }
 
+  private ParametersResolver getParameterResolverFromParameterValues(Map<String, Object> parameters) throws MuleException {
+    return getParameterResolverForParameterizedModel(parameters, oauthConfig.getDelegateConnectionProviderModel());
+  }
+
+  private ParametersResolver getParameterResolverForParameterizedModel(Map<String, Object> parameters,
+                                                                       ParameterizedModel parameterizedModel)
+      throws MuleException {
+    return getParameterResolverForParameterizedModelFromStaticValues(parameters,
```

Comment:
rename to `getParemetersResolverFromStaticValues`

Reply:
👍